Source/WebCore/page/TextAnimationTypes.h

```diff
@@ -45,6 +45,8 @@ struct TextAnimationData {
     TextAnimationType style;
     TextAnimationRunMode runMode;
     WTF::UUID unanimatedRangeUUID { WTF::UUID::emptyValue };
+    WTF::UUID sourceAnimationUUID { WTF::UUID::emptyValue };
```

Comment:
can these be ObjectIdentifiers?

Reply:
Everything runs on UUIDs right now, I don't want to change that at this point, but we're free to do a full changeover in the future!